Philips.CodeAnalysis.Common/DiagnosticId.cs

```diff
@@ -144,5 +144,6 @@ public enum DiagnosticId
 		AvoidUnusedToString = 2153,
 		AvoidUnlicensedPackages = 2155,
 		AvoidPkcsPaddingWithRsaEncryption = 2158,
+		TestTimeoutsClassShouldBeInternal = 2159,
```

Comment:
Let's use 2162 instead of 2159. 2159, 2160, and 2161 are claimed in other PRs. Remember to update the PH2159.md file name and contents as well.